products/satellite.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Red Hat Satellite Server
+category: server-app
+changelogTemplate: "https://access.redhat.com/documentation/en-us/red_hat_satellite/__RELEASE_CYCLE__/html/release_notes/index"
+releases:
+-   releaseCycle: "6.12"
+    eol: 2024-05-31
+    support: 2023-05-31
+    releaseDate: 2022-11-16
+    latest: "6.12.0"
+    latestReleaseDate: 2022-11-16
+-   releaseCycle: "6.11"
+    eol: 2024-01-31
+    support: 2022-11-30
+    releaseDate: 2022-07-05
+    latest: "6.11.4.1"
+    latestReleaseDate: 2022-11-17
+-   releaseCycle: "6.10"
+    eol: 2023-05-31
+    support: 2022-06-30
+    releaseDate: 2021-11-16
+    latest: "6.10.7.1"
+    latestReleaseDate: 2022-11-17
+-   releaseCycle: "6.9"
+    eol: 2022-11-30
+    support: 2021-11-30
+    releaseDate: 2021-11-16
+    latest: "6.9.10"
+    latestReleaseDate: 2022-11-17
+iconSlug: redhat
+permalink: /satellite
+alternate_urls:
+-   /rhsat
+releasePolicyLink: https://access.redhat.com/support/policy/updates/satellite
+activeSupportColumn: Full support
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Maintenance support
+discontinuedColumn: false
+versionCommand: yum info satellite
+
+---
+
+[Red Hat® Satellite](https://www.redhat.com/technologies/management/satellite) is an infrastructure management product specifically designed to keep 
+Red Hat Enterprise Linux® environments and other Red Hat infrastructure running efficiently, with security, and compliant with various standards.
+
+Red Hat Satellite Server is based on Open-Source technology.
+To name a few:
+- [Foreman](https://theforeman.org)
+- [Katello](https://theforeman.org/plugins/katello/)
+- [Pulp](https://pulpproject.org)
+- [Candlepin](https://www.candlepinproject.org)
```

Comment:
Not sure if this list is helpful to the average RH Sat user, who visits to check for support. Unless we provide a version/support mapping to these projects perhaps.

Could be dropped, imo.

Reply:
@heuwerk ping

Reply:
@captn3m0 You're right. Could you just delete it please?